101-privatelink-endpoint-sql/azuredeploy.json

```diff
@@ -0,0 +1,295 @@
+    "vnetName": "myVirtualNetwork",
+    "vnetAddressPrefix": "10.0.0.0/16",
+    "subnet1Prefix": "10.0.0.0/24",
+    "subnet1Name": "mySubnet",
+    "sqlServerName": "[concat('sqlserver', uniqueString(resourceGroup().id))]",
+    "databaseName": "[concat(variables('sqlServerName'),'/sample-db')]",
+    "privateEndpointName": "myPrivateEndpoint",
+    "privateDnsZoneName": "privatelink.database.windows.net",
+    "pvtendpointdnsgroupname": "[concat(variables('privateEndpointName'),'/mydnsgroupname')]",
+    "vmName": "[take(concat('myVm', uniqueString(resourceGroup().id)),15)]",    
+    "publicIpAddressName": "[concat(variables('vmName'), 'PublicIP' )]",
+    "networkInterfaceName": "[concat(variables('vmName'),'NetInt')]",
+    "osDiskType": "Standard_LRS"
+  },
+  "resources": [
+    {
+      "name": "[variables('sqlServerName')]",
+      "type": "Microsoft.Sql/servers",
+      "apiVersion": "2019-06-01-preview",
+      "location": "[parameters('location')]",
+      "tags": {
+        "displayName": "[variables('sqlServerName')]"
+      },
+      "kind": "v12.0",
+      "properties": {
+        "administratorLogin": "[parameters('administratorLogin')]",
+        "administratorLoginPassword": "[parameters('administratorLoginPassword')]",
+        "version": "12.0",
+        "publicNetworkAccess": "Disabled"
+      },
+      "resources": [
+      ]
+    },
+    {
+      "name": "[variables('databaseName')]",
+      "type": "Microsoft.Sql/servers/databases",
+      "apiVersion": "2019-06-01-preview",
+      "location": "[parameters('location')]",
+      "tags": {
+        "displayName": "[variables('databaseName')]"
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.Sql/servers', variables('sqlServerName'))]"
+      ],
+      "sku": {
+        "name": "Basic",
+        "tier": "Basic",
+        "capacity": 5
+      },
+      "properties": {
+        "collation": "SQL_Latin1_General_CP1_CI_AS",
+        "edition": "Basic",
+        "maxSizeBytes": 104857600,
+        "requestedServiceObjectiveName": "Basic",
+        "sampleName" : "AdventureWorksLT"
+      }
+    },
+
+    {
+      "apiVersion": "2018-10-01",
+      "type": "Microsoft.Network/virtualNetworks",
+      "name": "[variables('vnetName')]",
+      "location": "[parameters('location')]",
+      "properties": {
+        "addressSpace": {
+          "addressPrefixes": [
+            "[variables('vnetAddressPrefix')]"
+          ]
+        }
+      },
+      "resources": [
+        {
+          "apiVersion": "2018-10-01",
+          "type": "subnets",
+          "location": "[parameters('location')]",
+          "name": "[variables('subnet1Name')]",
+          "dependsOn": [
+            "[variables('vnetName')]"
+          ],
+          "properties": {
+            "addressPrefix": "[variables('subnet1Prefix')]",
+            "privateEndpointNetworkPolicies": "Disabled"
+          }
+        }
+      ]
+    },
+    {
+      "location": "[parameters('location')]",
```

Comment:
see the BP for ordering property 
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

Reply:
done in 05b1977